portfolio/src/main/java/com/google/sps/servlets/LoginStatusServlet.java

```diff
@@ -50,7 +50,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
             if (username == null) {
                 response.getWriter().println("{\"status\": \"True\", \"logoutUrl\": \"" + logoutUrl + "\", \"username\": \"null\"}");
             } else {
-                response.getWriter().println("{\"status\": \"True\", \"logoutUrl\": \"" + logoutUrl + "\"}");
+                response.getWriter().println("{\"status\": \"True\", \"logoutUrl\": \"" + logoutUrl + "\", \"username\": \"" + username + "\"}");
```

Comment:
I think it might be easier to make a separate class to store status, url, and username.

Reply:
Yes, this would make it much easier. I'll address it in a new pull request.